src/Symfony/Bundle/FrameworkBundle/Resources/config/validator.php

```diff
@@ -95,6 +96,12 @@
                 'alias' => NotCompromisedPasswordValidator::class,
             ])
 
+        ->set('validator.when', WhenValidator::class)
```

Comment:
If I don't miss anything, we need to remove this service definition from inside the `FrameworkExtension` in case the `WhenValidator` class does not exist (which can be the case when FrameworkBundle 6.2 is used with an older release of the Validator component).

Reply:
Done.